bin/index.js

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env node
+var DraftLog = require('draftlog').into(console).addLineListener(process.stdin)
+var SpeedTestNet = require('../')
+
+const DIM = '\033[2m'
```

Comment:
I avoid adding dependencies if they are not totally necessary. If you want, I can change to `chalk`, but would be cool to keep deps as low as possible (it's a goal that I keep to myself). But you are the boss ;)